tests/test_validate_outputs.py

```diff
@@ -409,7 +409,11 @@ def test_templates(self):
 
     def test_image_difference(self):
         """Test existence of image differences."""
-        self.check_pipetasks(["imageDifference"], self._num_exposures, self._num_exposures)
+        self.check_pipetasks(
+            ["subtractImages", "detectAndMeasureDiaSources"],
+            self._num_exposures,
+            self._num_exposures
+        )
         self.check_datasets(["goodSeeingDiff_differenceExp"], self._num_exposures_good_templates)
         self.check_datasets(["goodSeeingDiff_diaSrc_schema"], 1)
```

Comment:
One question: should this test case be split to test the outputs of `subtractImages` and `detectAndMeasureDiaSources` separately?

Reply:
I think combining them in one check is actually preferable, since we will want to pass the intermediate data products in memory in the future.